src/com/google/enterprise/adaptor/prebuilt/RegexFilter.java

```diff
@@ -208,21 +199,20 @@ private boolean foundInParams(Map<String, String> params) {
    * Conditionally adds a {@code Transmission-Decision} entry to the
    * {@code params Map}. The decision is based on settings of the
    * {@code key}, {@code pattern}, {@code decideOnMatch}, {@code decision},
-   * and {@code corpora} configuration variables (as discussed above).
+   * and {@code keyset} configuration variables (as discussed above).
    */
   @Override
   public void transform(Metadata metadata, Map<String, String> params) {
     boolean found;
-    switch (corpora) {
+    switch (keyset) {
       case METADATA:
         found = foundInMetadata(metadata);
         break;
       case PARAMS:
         found = foundInParams(params);
         break;
-      case METADATA_OR_PARAMS:
       default:
-        found = foundInMetadata(metadata) || foundInParams(params);
+        found = false;	// can't happen
```

Comment:
I feel like this should at least log a warning or even just throw AssertionError (the Google recommendation for enums), rather than just silently not match.